ucsmsdk/ucsmo.py

```diff
@@ -107,6 +107,27 @@ def _set_mo_prop_value(self, kwargs):
     def _is_unknown_property(self, prop):
         return prop not in self.prop_meta
 
+    def check_prop_match(self, **kwargs):
+        for prop_name in kwargs:
+            if prop_name not in self.prop_meta.keys():
```

Comment:
@parag-may4 

```
+            if prop_name not in self.prop_meta.keys():
```

is same as

```
+            if prop_name not in self.prop_meta:
```

for which an internal method already exists
```
    def _is_unknown_property(self, prop):                                       
    ¦   return prop not in self.prop_meta 
```